### PR TITLE
[POAE7-2702] Updata in function yaml-file to support in:str_list

### PR DESCRIPTION
**Pull request by haiweizh**
### What changes were proposed in this pull request?

Updata in function yaml-file and code to support in:str_list
Update velox test case code to support function mapping between presto and cider.

### Why are the changes needed?

Updata in function yaml-file to support in:str_list

### Does this PR introduce _any_ user-facing change?

No

### How was this patch tested?

Uts

### Which label does this PR belong to?

cider

### CODE REVIEW

**Comment by haiweizh** [comment]
Update code to solve problems encountered in performance test. Plz help to take a review. @yma11 @harborn @Ma-Jian1

File: cider-velox/test/expression/VeloxToCiderExpr.cpp [hunk 1]
```diff
@@ -168,6 +168,14 @@ std::shared_ptr<Analyzer::Expr> VeloxToCiderExprConverter::toCiderExpr(
...
     std::shared_ptr<const CallTypedExpr> vExpr,
     std::unordered_map<std::string, int> colInfo) const {
+  static const std::unordered_map<std::string, std::string> scalarMappings{
+      {"plus", "add"},
+      {"minus", "subtract"},
+      {"mod", "modulus"},
+      {"eq", "equal"},
+      {"neq", "not_equal"},
+      {"substr", "substring"},
+  };
```
Comment (yma11) [inline cider-velox/test/expression/VeloxToCiderExpr.cpp:178]: maybe this map should define inside FunctionLookupEngine?

Comment (haiweizh) [reply cider-velox/test/expression/VeloxToCiderExpr.cpp:179]: Done.

**Comment by haiweizh** [comment]
Hot fix. Plz help to take a review. @yma11 @jikunshang

**Review by jikunshang — DISMISSED** [review]

File: cider-velox/test/expression/VeloxToCiderExpr.cpp (hunk 1 above)
Comment (yma11) [reply cider-velox/test/expression/VeloxToCiderExpr.cpp:179]: should this map be removed here?

**Review by yma11 — DISMISSED** [review]

**Pull request closed without merging**